CEN-142 Add horizontal pod autoscaler for worker

michcio1234 opened the pull request:
and bump version (although it's not up-to-date with upstream repository)

michcio1234 committed:
Add horizontal pod autoscaler for worker

and bump version (although it's not up-to-date with upstream)

kayibal commented on dask/templates/dask-worker-autoscaler.yaml:
```diff
@@ -0,0 +1,20 @@
+    name: {{ template "dask.fullname" . }}-worker
+  minReplicas: {{ .Values.worker.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.worker.autoscaling.maxReplicas }}
+  targetCPUUtilizationPercentage: {{ .Values.worker.autoscaling.targetCPUUtilizationPercentage }}
```
I would add memory usage as it is more important.

michcio1234 committed:
Specify memory utilisation target

Is uses v2beta2 API version.
Weird things happen when you try to do `kubectl edit hpa` - it appears as
a v1 API resource and it doesn't show the whole `metrics` section.
But if you do `kubectl describe hpa`, it shows correctly.
